.github/workflows/autobuild.yml

```diff
@@ -2,7 +2,7 @@ name: Autobuild
 
 on:
   schedule:
-    - cron: '59 * * * *'
+    - cron: '01 05 * * *' # UTC, corresponds to 00:01 Central Time
```

Comment:
To be pedantic, if this is UTC, then it should vary between 23:01 and 00:01 (CST and CDT, respectively).

Reply:
good point, fixed.